### PR TITLE
Added supportsWiredMacAuth support for the AeroHIVE Branch Router 100

### PR DESCRIPTION
**Pull request by lzammit**

### CODE REVIEW

**Comment by cgx** [comment]
Please adjust the POD and the copyrights year.

File: lib/pf/Switch/AeroHIVE/BR100.pm [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package pf::Switch::AeroHIVE::BR100;
+
+=head1 NAME
+
+pf::Switch::AeroHIVE::AP - Object oriented module to access AP series via Telnet/SSH
```
Comment (extrafu) [inline lib/pf/Switch/AeroHIVE/BR100.pm]: Relevant?

File: lib/pf/Switch/AeroHIVE/BR100.pm [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+=head1 SYNOPSIS
+
+The pf::Switch::AeroHIVE::AP module implements an object oriented interface
```
Comment (extrafu) [inline lib/pf/Switch/AeroHIVE/BR100.pm]: Relevant?

**Comment by extrafu** [comment]
NEWS file entry!

File: lib/pf/Switch/AeroHIVE/BR100.pm [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+use Net::SNMP;
+use Try::Tiny;
+
+use base ('pf::Switch::AeroHIVE');
+use pf::config;
+use pf::Switch::constants;
+use pf::util;
+use pf::accounting qw(node_accounting_current_sessionid);
+use pf::node qw(node_attributes);
+use pf::util::radius qw(perform_coa perform_disconnect);
```
Comment (cgx) [inline lib/pf/Switch/AeroHIVE/BR100.pm]: You probably don't need to import so many modules. I would try:

``` perl
use strict;
use warnings;

use base ('pf::Switch::AeroHIVE');

use pf::config;
```

File: lib/pf/Switch/AeroHIVE/BR100.pm [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+=head1 STATUS
+
+This module is currently only a placeholder, see pf::Switch::AeroHIVE
+
+=cut
+
+use strict;
+use warnings;
+use Log::Log4perl;
+use Net::SNMP;
```
Comment (dwlfrth) [inline lib/pf/Switch/AeroHIVE/BR100.pm]: You don't need me

File: lib/pf/Switch/AeroHIVE/BR100.pm [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+use Try::Tiny;
```
Comment (dwlfrth) [inline lib/pf/Switch/AeroHIVE/BR100.pm]: You don't need me either

Comment (dwlfrth) [inline lib/pf/Switch/AeroHIVE/BR100.pm]: would looks great that way:
use strict;
use warnings;

use Log::Log4perl;

use base ('pf::Switch::AeroHIVE');

use pf::Switch::constants;

sub supportsWiredMacAuth { return $TRUE; }

**Pull request merged**